data/core/macros/abilities.cfg

```diff
@@ -266,6 +266,70 @@ Enemy units cannot see this unit while it is in deep water, except if they have
     [/hides]
 #enddef
 
+#define ABILITY_DISTRACT
+    [skirmisher]
+        id=distract
+        name= _ "distract"
+        female_name= _ "female^distract"
+        description= _ "This unit negates enemy Zones of Control around itself for allied units (but not for itself)."
+        affect_self=no
+        affect_allies=yes
+        [affect_adjacent]
+        [/affect_adjacent]
+    [/skirmisher]
+#enddef
+
+#define UTBS_ABILITY_HEALS
+    [heals]
+        value=4
+        id=healing
+        affect_allies=yes
+        name= _ "heals +4"
+        female_name= _ "female^heals +4"
+        description=  _ "Allows the unit to heal adjacent allied units at the beginning of our turn.
+
+A unit cared for by this healer may heal up to 4 HP per turn, or stop poison from taking effect for that turn.
+A poisoned unit cannot be cured of its poison by a healer, and must seek the care of a village or a unit that can cure."
+        affect_self=no
+        poison=slowed
+        [affect_adjacent]
+            [filter]
+                [not]
+                    status=dehydration_slowed_by_healer
+                [/not]
+            [/filter]
+        [/affect_adjacent]
+    [/heals]
+#enddef
```

Comment:
It would be better to add that into the normal heals ability. Then all kind of other units have the filter too, but it doesn't interfere there. I will on the other hand avoid duplicated code, and it will be one macro less.

Reply:
Okay, sounds good. I think the same can be done for cures ability.

Reply:
For the content of the macros, renaming the UTBS_HEALS to the "new" HEALS.

Though if you change the source code at the place of the "old" ability it will be visible in the diff what changed.

Reply:
I think of just removing the ability (giving elves the ABILITY_HEALS) or doing like @CelticMinstrel did. (Since I am pretty sure I don't understand you.)